plugins/woocommerce/src/Admin/Features/NewProductManagementExperience.php

```diff
@@ -28,6 +29,16 @@ public function __construct() {
 
 		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_styles' ) );
 		add_action( 'get_edit_post_link', array( $this, 'update_edit_product_link' ), 10, 2 );
+		if ( isset( $_GET['new-product-experience-disabled'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+			TransientNotices::add(
```

Comment:
Should we redirect removing `new-product-experience-disabled` from the URL so that this message does not appear if the user refreshes?

Reply:
Good idea! I add it in the commit [7e43808](https://github.com/woocommerce/woocommerce/pull/36544/commits/7e438083f7cc45d4ef2b2781eb21a559605de1c9)